README.md

```diff
@@ -16,6 +16,8 @@ includes all changes in the latest node-statsd and many additional changes, incl
 * asyncDistTimer
 * much more, including many bug fixes
 
+You can read about all changes in [the changelog](CHANGES.md).
+
```

Comment:
One of the most helpful files in the project as it's kept well up to date, so moving it earlier.  And removing two sections that just feel dated below.